### PR TITLE
Landing page and Menu UI

### PR DESCRIPTION
**Pull request by bhargavmodak**
## Landing page
- Mostly complete landing page
25-12-22   brave_2607
- Need discussion about images to add
- Add more info to contact section if needed
- Landing page is mostly responsive, needs dedicated tester

## Menu for options
image image

### CODE REVIEW

**Review by vishal-codes — CHANGES_REQUESTED** [review]
The contact section can be improved. 
1. You can add sumn like - "Want to report a bug or just wanna have a chat? Our inbox is always open, feel free to drop a text" and also add our twitter account. 
2. Add a back to top button and also a feedback button which will redirect to a feedback form(I've provided the editor access to u via email) 

Also, at the left most corner of navbar can u add the product's logo? doesn't need to spin.

**Comment by bhargavmodak** [comment]
image
image

File: client/src/LandingPage.js [hunk 1]
```diff
@@ -197,8 +216,8 @@ const LandingPage = () => {
                         width: '100%',
                         minHeight: '100vh',
                         padding: '4rem',
-                        background:
-                            'linear-gradient(116.82deg, #0288d1 0%, #1976d2 100%)',
+                        background: 'linear-gradient(116.82deg, rgba(2, 136, 209, 0.75) 0%, rgba(25, 118, 210, 0.75) 100%), url(/assets/landing-page-imgs/earth-digital.jpg)',
```
Comment (bhargavmodak) [inline client/src/LandingPage.js:219]: @vishal-codes You can change the image url and properties here.

**Comment by bhargavmodak** [comment]
## Complete Landing Page

https://user-images.githubusercontent.com/82528318/211069286-57a31ac9-836f-46ae-b8f3-ed66c4f1b623.mp4

### Changing the background picture

Kindly See : https://github.com/Evozone/dev-call/pull/58/commits/7caf0d02821866015b575cb1cf609de9634af92b#r1063650566

## Tests Added

### FeatureCard.test.js
- Cannot test left-right behavior because React works on a virtual DOM and Jest can only test DOM elements

File: client/src/LandingPage.js (hunk 1 above)
Comment (vishal-codes) [reply client/src/LandingPage.js:219]: Cool got it

**Review by vishal-codes — APPROVED** [review]
That's some great work!

**Pull request merged**